### PR TITLE
Part 2 only

### LINKED ISSUE #157: Assorted tasks
From @addisonr08.

- [x] Update team picture on the home page next to the paragraph about what team 1418 is. The picture is outdated (it's from the 2016 season).
- [x] Look at potential replacement video for the background of the home page, it's also from 2016. We have time lapse footage of the team building the robot, ask Zach for that if you want it. This isn't really a necessity, the current one is fine and it's hard to tell it's from 2016, but if you can find something which you think might work better feel free.
- [x] Make sure the members page is up to date. Add in people who are active, remove people who aren’t. Talk to Addison before removing anyone.
- [x] Update the image at the bottom of the sponsorship page, it's outdated (2016). 
- [x] Update the Outreach page with information about Erik's testifying in Richmond (maybe wait on this because he's not done yet) 
- [x] Work on changing the outreach page to be more accurate by changing the tenses of words from current to past in some cases. Ex. The “Support to FIRST Senior Mentor” section. We no longer currently do that, so it needs to change from current tense to past tense.
- [x] Make sure all information in the contact page is up to date.
- [x] Spacing on the top of the resources page seems kind of off, fix if possible
- [x] We will update sponsors at a later date as we learn who will be sponsoring us in the coming season
- [x] Replace outdated pictures of the 2016 robot where applicable with the 2017 robot, at least until we do the reveal of our new bot

Here are all the pictures from the team of this season, should be helpful. All are in glorious 4K, so keep file size in mind and reduce the resolution as needed, talk with me about that.
https://drive.google.com/drive/folders/0B93fACMelk1iaUJWejJwajZNQms?usp=sharing

### PR DESCRIPTION
**Pull request by NathanielHorne**
Fixes #157 part 2

### CODE REVIEW

File: _pages/team.html [hunk 1]
```diff
@@ -118,6 +130,60 @@ <h5>Mira Vega</h5>
...
+            <img src="/assets/img/team/placeholder.png">
+            <h5>Connor Floyd</h5>
+            <!---TODO: add details---->
+            <p>Connor is a programmer. He has helped to edit our online guides and to code the website.</p>
+        </div>
+        <div>
+            <img src="/assets/img/team/placeholder.png">
+            <h5>Connor Mills</h5>
+            <!---TODO: add details---->
+            <p>Bumpers. Mechanic</p>
```
Comment (ErikBoesen) [inline _pages/team.html]: We need to have bios here, not just individual words.

File: _pages/team.html [hunk 2]
```diff
@@ -118,6 +130,60 @@ <h5>Mira Vega</h5>
             <h5>Keenan Hom</h5>
             <p>Keenan is a mechanic.</p>
         </div>
+        <div>
+            <img src="/assets/img/team/placeholder.png">
+            <h5>Connor Floyd</h5>
+            <!---TODO: add details---->
+            <p>Connor is a programmer. He has helped to edit our online guides and to code the website.</p>
```
Comment (ErikBoesen) [inline _pages/team.html]: Conor

File: _pages/team.html [hunk 3]
```diff
@@ -118,6 +130,60 @@ <h5>Mira Vega</h5>
...
+        <div>
+            <img src="/assets/img/team/placeholder.png">
+            <h5>Fernanda Molina</h5>
+            <!---TODO: add details---->
+            <p>Programming</p>
+        </div>
+        <div>
+            <img src="/assets/img/team/placeholder.png">
+            <h5>Blaine Morse</h5>
+            <!---TODO: add details---->
```
Comment (ErikBoesen) [inline _pages/team.html]: This is how you do HTML comments:
```html
<!-- Comment -->
```
So, you don't need this many hyphens.

File: _pages/team.html [hunk 4]
```diff
@@ -118,6 +130,60 @@ <h5>Mira Vega</h5>
...
+            <p>Programming</p>
+        </div>
+        <div>
+            <img src="/assets/img/team/nathaniel-horne.png">
+            <h5>Johnny Assel</h5>
```
Comment (ErikBoesen) [inline _pages/team.html]: Didn't know your name is Johnny :)

By the way, there's already a file which you can rename to `nathaniel-horne.png`. It's called `ben-rice.png`. Probably better to rename the file than make a new one.

File: _pages/team.html [hunk 5]
```diff
@@ -98,6 +98,18 @@ <h5>Carolyn Baek</h5>
...
+        <div>
+            <img src="/assets/img/team/nikita-solovyov.png">
+            <h5>Nikita Solovyov</h5>
+            <!---TODO: add detials.--->
+            <p></p>
+        </div>
+        <div>
+            <img src="/assets/img/team/placeholder.png">
+            <h5>Anuragi Thapliya</h5>
+            <!---TODO: add detials.--->
```
Comment (ErikBoesen) [inline _pages/team.html]: We already have Anuragi, Carolyn, and maybe a couple others. Check if there are any others who are already on the website: http://1418.team/team

File: _pages/team.html [hunk 6]
```diff
@@ -118,6 +130,60 @@ <h5>Mira Vega</h5>
...
+        </div>
+        <div>
+            <img src="/assets/img/team/placeholder.png">
+            <h5>Johnny Assel</h5>
+            <!---TODO: add details---->
+            <p>Johnny works with strategy and scoutting. His proficiency with scoutting data has been invaluable to the team.</p>
+        </div>
+        <div>
+            <img src="/assets/img/team/jakub-leski.png">
+            <h5>Jakub Leski</h5>
```
Comment (ErikBoesen) [inline _pages/team.html]: Jakub's already also there.

File: _pages/team.html [hunk 7]
```diff
@@ -118,6 +130,60 @@ <h5>Mira Vega</h5>
...
+        </div>
+        <div>
+            <img src="/assets/img/team/nathaniel-horne.png">
+            <h5>Johnny Assel</h5>
+            <!---TODO: add details---->
+            <p>Nathaniel is a programmer. He is the de facto editor of the website itself.</p>
+        </div>
+        <div>
+            <img src="/assets/img/team/placeholder.png">
+            <h5>Johnny Assel</h5>
```
Comment (ErikBoesen) [inline _pages/team.html]: His last name is Asel.

File: _pages/team.html [hunk 8]
```diff
@@ -118,6 +130,60 @@ <h5>Mira Vega</h5>
...
+            <!---TODO: add details---->
+            <p>Johnny works with strategy and scoutting. His proficiency with scoutting data has been invaluable to the team.</p>
```
Comment (ErikBoesen) [inline _pages/team.html]: "Scouting."

Also, it hasn't really been helpful yet... not that it won't be in the future.

File: _pages/team.html [hunk 9]
```diff
@@ -118,6 +130,60 @@ <h5>Mira Vega</h5>
...
+        </div>
+        <div>
+            <img src="/assets/img/team/jakub-leski.png">
+            <h5>Jakub Leski</h5>
+            <!---TODO: add details---->
+            <p>Bumpers</p>
+        </div>
+        <div>
+            <img src="/assets/img/team/placeholder.png">
+            <h5>Joseph harpring</h5>
```
Comment (ErikBoesen) [inline _pages/team.html]: Who is this?

File: _pages/team.html [hunk 10]
```diff
@@ -118,6 +130,60 @@ <h5>Mira Vega</h5>
...
+            <!---TODO: add details---->
+            <p></p>
+        </div>
+        <div>
+            <img src="/assets/img/team/carolyn-baek.png">
+            <h5>Carolyn Baek</h5>
+            <!---TODO: add details---->
```
Comment (ErikBoesen) [inline _pages/team.html]: I don't think we need these comments everywhere.

**Comment by ErikBoesen** [comment]
Also, I'd recommend cropping and/or scaling down the team photo. 6000x4000 images will take forever to load.

File: _includes/footer.html [hunk 11]
```diff
@@ -14,6 +14,9 @@ <h5>We love our sponsors!</h5>
                 <td><a href="https://erikboesen.com/ebt"><img src="/assets/img/sponsors/ebt.svg"></a></td>
             </tr>
             <tr>
+                <td><a href="http://www.fcedf.org/"><img src="/assets/img/sponsors/fcef.svg"></a></td>
```
Comment (ErikBoesen) [inline _includes/footer.html]: Don't mean to be a downer but uhh... these three are already on the website.

File: Newrobot.svg [hunk 12]
```diff
@@ -0,0 +1,26 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"  width="400" height="400">
```
[the remark this answers is not in the document]
Comment (twinters007) [reply Newrobot.svg]: Also github says it's invalid if that means anything

File: _includes/footer.html [hunk 13]
```diff
@@ -11,9 +11,15 @@ <h5>We love our sponsors!</h5>
             <tr>
                 <td><a href="http://www.boozallen.com/"><img src="/assets/img/sponsors/booz-allen-hamilton.svg"></a></td>
                 <td><a href="http://www.setty.com/"><img class="sponsor-large" src="/assets/img/sponsors/setty.svg"></a></td>
-                <td><a href="https://erikboesen.com/ebt"><img src="/assets/img/sponsors/ebt.svg"></a></td>
+                <td><a href="http://erikboesen.com/vektre"><img class="sponsor-large" src="/assets/img/sponsors/vektre.svg"></a></td>
```
Comment (twinters007) [inline _includes/footer.html]: this website just...doesn't exist

**Pull request closed without merging**